lib/domain.js

```diff
@@ -16,28 +18,28 @@ const internals = {
 exports.analyze = function (domain, options = {}) {
 
     if (typeof domain !== 'string') {
-        throw new Error('Invalid input: domain must be a string');
+        throw new Error(errorsByCode.DOMAIN_BE_STRING);
     }
 
     if (!domain) {
-        return { error: 'Domain must be a non-empty string' };
+        return { error: errorsByCode.DOMAIN_NON_EMPTY_STRING };
```

Comment:
If we are going to do this, we should add actual codes. E.g. `{ error: errorsByCode.DOMAIN_NON_EMPTY_STRING, code: 'DOMAIN_NON_EMPTY_STRING' }`

Reply:
Ok